src/Directory.Build.props

```diff
@@ -63,7 +63,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.App" Version="$(MicrosoftNETCoreAppVersion)" Condition="'$(TargetFramework)' == 'netcoreapp2.1'" />
+    <PackageReference Include="Microsoft.NETCore.App" Version="$(MicrosoftNETCoreAppVersion)" Condition="'$(TargetFramework)' == 'netcoreapp2.1'" PrivateAssets="All"/>
```

Comment:
Should it be conditioned on `DisableImplicitFrameworkReferences` which is the flag that also tells the SDK to not add an implicit one? https://github.com/AndyGerlicher/msbuild/commit/e735b93886f1ac64b82eec2684cedcdd6892ef5f#diff-9002cc0a94d7643cb073436b973a63ebR143

Reply:
No, I don't think so: this should only be added when that's set.